pygpu/gpuarray.pyx

```diff
@@ -2229,13 +2229,40 @@ cdef class GpuArray:
         def __get__(self):
             return self.ga.offset
 
+    property data:
+        """Return a pointer to the raw OpenCL buffer object.
+
+        This will fail for arrays that have an offset.
```

Comment:
It's not possible to just add the offset in the OpenCL case.

I chose the name `data` to help compatibility with PyOpenCL, but the exception would be there regardless.